src/hooks.tsx

```diff
@@ -185,3 +187,14 @@ export const useIsInViewport = (
 
     return isInViewport;
 };
+
+export const useContainerType = (): ContainerType => {
+    const containerType = React.useContext(ContainerTypeContext);
+    if (!containerType) {
+        throw Error(
```

Comment:
Are we 100% sure that we want this to throw an error, there's no possibility of using one of the mistica atoms that use containerType without having this context?

Reply:
mmm, the error is raised only when the hook is used on a component not placed on a responsive layout, nor a grid layout. By now, only the carousels are using it.

Question @yceballost: if not in a responsive layout nor grid layout, what value should arrive to the component via this hook ? maybe depending on the screen media queries ? I can do that if you think thats the way to go:
- mobile for mobile screen
- tablet for tablet
- desktop and wide desktop for desktop and large desktop respectively. 

WDYT ?

Reply:
for me it is ok!

In this way, the actual behavior keeps the same behavior, only changing when it is on a responsive or grid layout and the container is less than the mq defined, right?